wlct/tournaments.py

```diff
@@ -2422,12 +2422,19 @@ def finish_game(self):
         self.is_finished = True
 
         self.current_state = "Finished"
+        self.tournament = find_tournament_by_id(self.tournament.id)
         log_tournament("Finish Game/Game Entry: {}".format(self.teams), self.tournament)
         team1 = self.teams.split('.')[0]
         team2 = self.teams.split('.')[1]
 
         tournament_team1 = get_team_by_id(self.tournament, int(team1))
         tournament_team2 = get_team_by_id(self.tournament, int(team2))
+
```

Comment:
Please add a comment...."if this tournament is actually a sub-tournament, look it up like this"

Reply:
Done